exonum/src/events/noise/wrapper.rs

```diff
@@ -134,17 +138,13 @@ impl NoiseWrapper {
 
     fn read(&mut self, input: &[u8], len: usize) -> Result<(usize, Vec<u8>), NoiseError> {
```

Comment:
(Slightly) off-topic: isn't it possible to guess the length of the output from the length of the input (like, the `input.len()` is the upper bound)? I'd like to see a signature here like

```rust
fn read(&mut self, input: &[u8]) -> Result<Vec<u8>, NoiseError>;
```

since this one looks too close to C for (at least my) comfort.